### ISSUE TITLE
Unable to download 0.13.X.X versions

### ISSUE THREAD

**Issue by Antonio231102** [MCMrARM/mc-w10-version-launcher#257]
Upon trying to download versions 0.13.0.0, 0.13.1.0, or 0.13.2.0, I am greeted with the message below. It seems like these versions are no longer available on Microsoft's servers.

![image](https://github.com/MCMrARM/mc-w10-version-launcher/assets/29684799/5711961a-873f-4464-9643-72e4fa177ae5)

If you happen to have the .appx files for these versions, would it be possible for you to upload them somewhere and update their links in the launcher?

**Comment by dktapps** [comment]
No, I'm afraid we can't do that. The launcher skirts around piracy concerns by downloading the game from MS servers, which require authentication using your XBL account. 

You can import appxes if you find them on the internet, but that's a risk for you to consider.

**Comment by Antonio231102** [comment]
No worries, that's totally understandable. Thank you for the quick response.

**Comment by villagervr** [comment]
hellllllo

**Comment by villagervr** [comment]
### 

> 

> 
FOOK 

1. > 

1. [ ] `RRFGRERGTEERER
[Uploading MCLaunc
[Uploading Sussy Schoolgrounds.zip…]()
her.zip…]()

- `

1.  @_
[Uploading MCLauncher.zip…]()
>_ 

> 

> RRR

> FIUK

> _**efghjhEEEEEEEEEE**ERHJKJHTRRFFFFFFF****_
